packages/gherkindoc/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gherkindoc",
```

Comment:
Can't use this NPM name, it's already reserved.

Reply:
RIP